src/coreclr/nativeaot/System.Private.Reflection.Core/src/System/Reflection/Runtime/MethodInfos/CustomMethodInvoker.cs

```diff
@@ -53,8 +53,30 @@ public sealed override Delegate CreateDelegate(RuntimeTypeHandle delegateType, o
         {
             if (_thisType.IsConstructedGenericType && _thisType.GetGenericTypeDefinition() == typeof(Nullable<>))
             {
-                // Desktop compat: MethodInfos to Nullable<T> methods cannot be turned into delegates.
-                throw new ArgumentException(SR.Arg_DlgtTargMeth);
+                if (isOpen)
+                {
+                    return DynamicDelegateAugments.CreateObjectArrayDelegate(Type.GetTypeFromHandle(delegateType),
```

Comment:
This won't work for methods that take refs, byref-like types, etc. Is that right?

Reply:
Yes, but Nullable doesn't have any of those. This only works for the following methods:

https://github.com/dotnet/corert/blob/24f1edc938af9954eac9a3385958787ca64956d5/src/System.Private.Reflection.Core/src/System/Reflection/Runtime/MethodInfos/CustomMethodMapper.Nullable.cs#L28-L96

Reply:
ah ok, I thought that it is for more than that.